.bundle/config

```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_WITHOUT: "production"
```

Comment:
Could you remove this file from the commit? 

Since I recently had to look up how to do this, it's
```
git rm --cached .bundle/config
git commit --amend
git push -f origin <branch>
```

Going the `--amend` route is a personal preference as it keeps my commit history down to a single commit on the branch, which prevents the need to do interactive rebasing down the road.

Reply:
This is very valuable info to keep on hand. Thanks @leila-alderman!